### PR TITLE
Redis

### PR DESCRIPTION
**Pull request by amerkul**

### CODE REVIEW

File: src/main/java/com/example/auction/config/RedisConfig.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        RedisStandaloneConfiguration redisStandaloneConfig = new RedisStandaloneConfiguration();
+        redisStandaloneConfig.setHostName(this.redisHost);
+        redisStandaloneConfig.setPort(this.redisPort);
+        return new LettuceConnectionFactory(redisStandaloneConfig);
+    }
+
+    @Bean
+    public ReactiveRedisOperations<String, Object> redisOperations(ReactiveRedisConnectionFactory reactiveRedisConnectionFactory) {
+        Jackson2JsonRedisSerializer<Object> serializer = new Jackson2JsonRedisSerializer<>(Object.class);
+
```
Comment (h1alexbel) [inline src/main/java/com/example/auction/config/RedisConfig.java]: empty lines detected

File: src/main/java/com/example/auction/config/RedisConfig.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        RedisSerializationContext.RedisSerializationContextBuilder<String, Object> builder =
+                RedisSerializationContext.newSerializationContext(new StringRedisSerializer());
+
+        RedisSerializationContext<String, Object> context = builder.value(serializer).hashValue(serializer)
+                                                                   .hashKey(serializer).build();
+
```
Comment (h1alexbel) [inline src/main/java/com/example/auction/config/RedisConfig.java]: empty lines in your method means that this method is too big

File: src/main/java/com/example/auction/kafka/consumer/ReactiveKafkaConsumer.java [hunk 3]
```diff
@@ -21,12 +22,17 @@ public class ReactiveKafkaConsumer {
...
     private final ReactiveProducerService producerService;
+    private RedisService redisService;
     private final AuctionService service;
 
     @PostConstruct
     public void init() {
         consumerTemplate.receive()
                         .subscribe(record -> {
+                            redisService.set(
+                                    record.value().getId().toString(), record.value().getId().toString(), record.value()
```
Comment (h1alexbel) [inline src/main/java/com/example/auction/kafka/consumer/ReactiveKafkaConsumer.java]: line is too long

File: src/main/java/com/example/auction/config/RedisConfig.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    @Bean
+    public ReactiveRedisOperations<String, Object> redisOperations(ReactiveRedisConnectionFactory reactiveRedisConnectionFactory) {
+        Jackson2JsonRedisSerializer<Object> serializer = new Jackson2JsonRedisSerializer<>(Object.class);
+
+        RedisSerializationContext.RedisSerializationContextBuilder<String, Object> builder =
+                RedisSerializationContext.newSerializationContext(new StringRedisSerializer());
+
+        RedisSerializationContext<String, Object> context = builder.value(serializer).hashValue(serializer)
```
Comment (h1alexbel) [inline src/main/java/com/example/auction/config/RedisConfig.java]: @amerkul Why context's value is represented by the `Object`? It should be your `AccountCreatedEvent`, not an object. Maybe I miss something.

Comment (amerkul) [reply src/main/java/com/example/auction/config/RedisConfig.java]: I agree with you.

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**